pkg/kudoctl/util/repo/index.go

```diff
@@ -162,6 +151,38 @@ func (i *IndexFile) WriteFile(fs afero.Fs, file string) (err error) {
 	return i.Write(f)
 }
 
+func (i *IndexFile) validate() error {
+	if i.APIVersion == "" {
+		return errors.New("no API version defined")
+	}
+
+	for _, packageVersions := range i.Entries {
+		for _, packageVersion := range packageVersions {
+			if packageVersion.Name == "" {
+				return errors.New("a package is missing a name")
```

Comment:
I removed this validation, because it doesn't make sense at this point. `Name` is already used as key in `i.Entries`. Having an empty string as name would have caused a panic earlier.